### PR TITLE
#23

### PR DESCRIPTION
**Pull request by strzalkom**
Feature/admin users view

### CODE REVIEW

File: src/main/java/com/sda/Warehouse/configs/InitialDataConfig.java [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (RafalCiolek) [inline src/main/java/com/sda/Warehouse/configs/InitialDataConfig.java]: useless line

File: src/main/java/com/sda/Warehouse/controllers/DevFrontController.java [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (RafalCiolek) [inline src/main/java/com/sda/Warehouse/controllers/DevFrontController.java]: useless line

File: src/main/java/com/sda/Warehouse/controllers/DevFrontController.java [hunk 3]
```diff
@@ -19,3 +20,4 @@ public String login() {
```
Comment (RafalCiolek) [inline src/main/java/com/sda/Warehouse/controllers/DevFrontController.java]: useless lines

File: src/main/java/com/sda/Warehouse/controllers/UserController.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+ * Created by strzalkom on 04.09.2017.
+ */
+
+
+@Controller
+@RequestMapping(path = "/users")
+public class UserController {
+
+    @Autowired
+    private JpaUserRepository userRepository;
```
Comment (RafalCiolek) [inline src/main/java/com/sda/Warehouse/controllers/UserController.java]: use constructor injection :)

File: src/main/java/com/sda/Warehouse/models/CreationStatus.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package com.sda.Warehouse.models;
+
+public class CreationStatus {
+    private boolean status;
+
+    private String message;
+
```
Comment (RafalCiolek) [inline src/main/java/com/sda/Warehouse/models/CreationStatus.java]: useless line

File: src/main/java/com/sda/Warehouse/models/CreationStatus.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+public class CreationStatus {
+    private boolean status;
+
+    private String message;
+
+
+    CreationStatus() {
+    }
+
+    public boolean isStatus() {
```
Comment (RafalCiolek) [inline src/main/java/com/sda/Warehouse/models/CreationStatus.java]: use lombok

File: src/main/java/com/sda/Warehouse/models/CreationStatusFactory.java [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+/**
+ * Created by lenovo on 04.09.2017.
+ */
+
+
+@Component
+public class CreationStatusFactory {
+
+    public com.sda.Warehouse.models.CreationStatus createSuccessStatus(String message) {
```
Comment (RafalCiolek) [inline src/main/java/com/sda/Warehouse/models/CreationStatusFactory.java]: import this package.

File: src/main/java/com/sda/Warehouse/models/CreationStatusFactory.java [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+        return createStatus(true, message);
+    }
+
+    public com.sda.Warehouse.models.CreationStatus createFailureStatus(String message) {
```
Comment (RafalCiolek) [inline src/main/java/com/sda/Warehouse/models/CreationStatusFactory.java]: import this package (will be shorter)

File: src/main/java/com/sda/Warehouse/models/CreationStatusFactory.java [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+        return createStatus(false, message);
+    }
+
+    private com.sda.Warehouse.models.CreationStatus createStatus(boolean status, String message) {
```
Comment (RafalCiolek) [inline src/main/java/com/sda/Warehouse/models/CreationStatusFactory.java]: import this package (will be shorter)

File: src/main/java/com/sda/Warehouse/models/User.java [hunk 10]
```diff
@@ -1,54 +1,103 @@
...
+
+    @Size(min = 1, max = 15)
+    @NotNull
+    private String role;
+
+    @Setter
+    @NotNull
+    private boolean isActive;
+
+    public String getFirstName() {
```
Comment (RafalCiolek) [inline src/main/java/com/sda/Warehouse/models/User.java]: use lombok

File: src/main/java/com/sda/Warehouse/repositories/JpaProductRepository.java [hunk 11]
```diff
@@ -16,3 +17,4 @@
     Page<Product> findByNameIgnoreCaseContainingOrDescriptionIgnoreCaseContaining(String name, String description, Pageable pageable);
```
Comment (RafalCiolek) [inline src/main/java/com/sda/Warehouse/repositories/JpaProductRepository.java]: This record is unreadable (too long), replace it in SQL query by @Query annotation

File: src/main/resources/templates/fragments/head.html [hunk 12]
```diff
@@ -1,3 +1,20 @@
+<<<<<<< HEAD
```
Comment (RafalCiolek) [inline src/main/resources/templates/fragments/head.html]: resolve conflicts

File: src/main/resources/templates/fragments/header.html [hunk 13]
```diff
@@ -1,3 +1,38 @@
+<<<<<<< HEAD
```
Comment (RafalCiolek) [inline src/main/resources/templates/fragments/header.html]: resolve conflicts

File: src/main/resources/templates/index.html [hunk 14]
```diff
@@ -1,3 +1,65 @@
+<<<<<<< HEAD
```
Comment (RafalCiolek) [inline src/main/resources/templates/index.html]: resolve conflicts

File: src/main/resources/templates/productPage.html [hunk 15]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Product</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```
Comment (RafalCiolek) [inline src/main/resources/templates/productPage.html]: replace by object:head

File: src/main/resources/templates/users.html [hunk 16]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8"/>
+    <title>Users</title>
```
Comment (RafalCiolek) [inline src/main/resources/templates/users.html]: is it work without css and js?

File: src/main/java/com/sda/Warehouse/configs/InitialDataConfig.java (hunk 1 above)
Comment (strzalkom) [reply src/main/java/com/sda/Warehouse/configs/InitialDataConfig.java]: done

File: src/main/java/com/sda/Warehouse/controllers/DevFrontController.java (hunk 2 above)
Comment (strzalkom) [reply src/main/java/com/sda/Warehouse/controllers/DevFrontController.java]: done

File: src/main/java/com/sda/Warehouse/controllers/DevFrontController.java (hunk 3 above)
Comment (strzalkom) [reply src/main/java/com/sda/Warehouse/controllers/DevFrontController.java]: done

File: src/main/java/com/sda/Warehouse/models/CreationStatus.java (hunk 5 above)
Comment (strzalkom) [reply src/main/java/com/sda/Warehouse/models/CreationStatus.java]: done

File: src/main/resources/templates/fragments/head.html (hunk 12 above)
Comment (strzalkom) [reply src/main/resources/templates/fragments/head.html]: done

File: src/main/resources/templates/fragments/header.html (hunk 13 above)
Comment (strzalkom) [reply src/main/resources/templates/fragments/header.html]: done

File: src/main/resources/templates/index.html (hunk 14 above)
Comment (strzalkom) [reply src/main/resources/templates/index.html]: done

File: src/main/java/com/sda/Warehouse/models/CreationStatusFactory.java (hunk 7 above)
Comment (strzalkom) [reply src/main/java/com/sda/Warehouse/models/CreationStatusFactory.java]: done

File: src/main/java/com/sda/Warehouse/models/CreationStatusFactory.java (hunk 8 above)
Comment (strzalkom) [reply src/main/java/com/sda/Warehouse/models/CreationStatusFactory.java]: done

File: src/main/java/com/sda/Warehouse/models/CreationStatusFactory.java (hunk 9 above)
Comment (strzalkom) [reply src/main/java/com/sda/Warehouse/models/CreationStatusFactory.java]: done

File: src/main/java/com/sda/Warehouse/models/User.java (hunk 10 above)
Comment (strzalkom) [reply src/main/java/com/sda/Warehouse/models/User.java]: done

**Comment by lukaszwasielewski** [comment]
resolve conflicts pls

**Pull request closed without merging**